drivers/net/phy/sfp.c

```diff
@@ -318,6 +320,15 @@ static unsigned int sfp_gpio_get_state(struct sfp *sfp)
 			state |= BIT(i);
 	}
 
+	/* Trivial debounce. When no state change is detected, wait for up to
+	 * a limited bound time interval for the signal state to settle.
+	 */
+	if (state == sfp->state && repeat > 0) {
```

Comment:
What do mainline developers say about this issue? Is this workaround acceptable?

Reply:
See https://lore.kernel.org/netdev/YyA1sfZRuoka8yhl@shell.armlinux.org.uk/T/#t .

I think Russell's comment should not be too hard to address.